docs/dml-of-carbondata.md

```diff
@@ -43,6 +43,7 @@ CarbonData DML statements are documented here,which includes:
   **NOTE**:
     * Use 'file://' prefix to indicate local input files path, but it just supports local mode.
     * If run on cluster mode, please upload all input files to distributed file system, for example 'hdfs://' for hdfs.
+    * Each load creates new segment folder and manages the folder through tablestatus file.
```

Comment:
since this content is already present in Segment Management. no need to add it

Reply:
ok. removed.